### ISSUE TITLE
Class that corresponds to each color

### ISSUE THREAD

**Issue by nefelipk** [CSAILVision/semantic-segmentation-pytorch#154]
Hello,

@hangzhaomit thanks for the Pytorch implementation on ADE20K dataset!
I wonder if there is a way to know which classes correspond to each color in the .png image we get as a result. Would you be so kind to provide me with any information about it? 

Thank you in advance

**Comment by shlykov2** [comment]
https://github.com/CSAILVision/semantic-segmentation-pytorch/blob/master/data/object150_info.csv
"idx"

**Comment by nefelipk** [comment]
Thanks @shlykov2! I can see, for example, that idx "1" corresponds to class "wall"... but how can i know the color in the image that corresponds to them?

**Comment by shlykov2** [comment]
load img with scipy imread (as i did, idk about other libraries) astype "uint8" and check values there. 1's, for instance, will stand for wall :) and etc

**Comment by nefelipk** [comment]
I load the img but I don't seem to understand how to use the mask in order to get the values I need. I am new to neural nets and matlab. Can you give me any other tips?

**Comment by harishreddy05** [comment]
Hope this helps,

[https://docs.google.com/spreadsheets/d/1se8YEtb2detS7OuPE86fXGyD269pMycAWe2mtKUj2W8/edit?usp=sharing](url)

**Comment by nefelipk** [comment]
Yes great! That is exactly what I was looking for!
Thank you very much!

**Comment by hangzhaomit** [comment]
> Hope this helps,
> 
> https://docs.google.com/spreadsheets/d/1se8YEtb2detS7OuPE86fXGyD269pMycAWe2mtKUj2W8/edit?usp=sharing

Thank you! Can I share this Google Doc with everyone?

**Comment by harishreddy05** [comment]
> > Hope this helps,
> > https://docs.google.com/spreadsheets/d/1se8YEtb2detS7OuPE86fXGyD269pMycAWe2mtKUj2W8/edit?usp=sharing
> 
> Thank you! Can I share this Google Doc with everyone?

Yeah sure!
If you think it will be helpful to others, you can even update the repo with this Google Sheet.
